README.md

```diff
@@ -124,12 +124,40 @@ To truncate data before re-loading it run
 - Create and seed using the following scripts:
 
         python -m scripts.{load_config_script}
+### Build with Paketo
+
+[Pack](https://buildpacks.io/docs/tools/pack/cli/pack_build/)
+
+[Paketo buildpacks](https://paketo.io/)
+
+```pack build <name your image> --builder paketobuildpacks/builder:base```
+
+Example:
+
+```
+[~/work/repos/funding-service-design-fund-store] pack build paketo-demofsd-app --builder paketobuildpacks/builder:base
+***
+Successfully built image paketo-demofsd-app
+```
+
+You can then use that image with docker to run a container
+
+```
+docker run -d -p 8080:8080 --env PORT=8080 --env FLASK_ENV=dev paketo-demofsd-app
+```
+
```

Comment:
Is it worth mentioning that you need to set SENTRY_DSN and GITHUB_SHA here?

Reply:
Quite right, might as well make them all consistent